lib/remap-tests/src/main.rs

```diff
@@ -10,12 +11,15 @@ fn main() {
         .unwrap_or_default();
 
     let mut failed_count = 0;
-    let tests = fs::read_dir("tests").expect("dir exists");
+    for entry in glob("tests/**/*.vrl").expect("valid pattern") {
+        let path = match entry {
+            Ok(path) => path,
+            Err(_) => continue,
+        };
 
-    for file in tests {
-        let mut test = Test::new(file.expect("file").path());
+        let mut test = Test::new(path);
 
-        print!("{:.<60}", test.name);
+        print!("{:.<70}", test.name);
 
         if test.skip {
             println!("{}", Colour::Yellow.bold().paint("SKIPPED"));
```

Comment:
I would argue that if we skip the test, we don't need to carry on and run the test.

Reply:
I'd like to. I used this to support `--verbose`, which prints the output of each run without showing a diff, regardless whether the test was skipped or not.

This helps (me) during debugging.